core/src/modules/ops/property.ts

```diff
@@ -134,7 +134,7 @@ export namespace PropertyOps {
     for (const key in ruleOptions) {
       const mustacheString = ruleOptions[key];
       const mustacheVariables: string[] = Mustache.parse(mustacheString)
-        .filter((chunk) => chunk[0] === "name")
+        .filter((chunk) => chunk[0] === "&")
```

Comment:
This is the missing piece!  When we use the triple bracket, `Mustache.parse`'s response looks different.  The ampersand tells mustache to render the value unescaped.

Reply:
That's 🍌s

Some notes here - https://stackoverflow.com/questions/22910428/mustache-globally-disable-html-escaping/22910523

Reply:
Interesting!  I had to dig around in their github repo to find this rule 😅